docs/manuals/characterization/stark_experiment.rst

```diff
@@ -169,7 +169,7 @@ by a variant of the Hahn-echo pulse sequence [5]_.
         "formatter.label_offset.pulse_name": 0.1,
         "formatter.text_size.annotate": 14,
     }
-    ram_x_schedule.draw(time_range=(0, 1600), style=IQXSimple(**opt), backend=backend)
+    ram_x_schedule.draw(time_range=(0, 1600), style=IQXSimple(**opt))
```

Comment:
this backend (switched to V2) is removed because of https://github.com/Qiskit/qiskit/pull/12608